src/js/ripple/remote.js

```diff
@@ -515,20 +515,13 @@ Remote.prototype.disconnect = function(callback) {
  */
 
 Remote.prototype._handleMessage = function(message, server) {
-  try {
-    message = JSON.parse(message);
-
-  /*eslint-disable no-empty*/
-  } catch (e) {
-    // TODO: why are we suppressing this?
-    console.error(e);
-  }
-  /*eslint-enable no-empty*/
-
   if (!Remote.isValidMessage(message)) {
```

Comment:
Oops, I misread this: https://github.com/ripple/ripple-lib/pull/328/files#diff-d6c2439c9363ff2c9f9f89fe2a857808R274